### ISSUE TITLE
Failure on Package Update

### ISSUE THREAD

**Issue by thomascrha** [davestephens/ansible-nas#13]
When I set `keep_packages_updated = true` in my `all.yml` I received this error - 

```
TASK [Upgrade all packages] ******************************************************************************************************************************************************
fatal: [localhost]: FAILED! => {"changed": false, "msg": "Unable to find APTITUDE in path. Please make sure to have APTITUDE in path or use 'force_apt_get=True'"}

PLAY RECAP ***********************************************************************************************************************************************************************
localhost                  : ok=15   changed=0    unreachable=0    failed=1
```

SOLUTION:

After then running `apt install aptitude` it worked. So I'm thinking of adding in a condition that if this setting is true check if aptitude is installed, if not install it. I've got no idea if this is doable - I very new to ansible - but I thought this would be a great start - hopefully a PR soon.

**Comment by davestephens** [comment]
What version of Ubuntu is this on? Please can you run `cat /etc/lsb_release` and copy the output?

**Comment by thomascrha** [comment]
```
tcrha@hopenas:~$ cat /etc/lsb_release
cat: /etc/lsb_release: No such file or directory
tcrha@hopenas:~$ lsb_release -a
No LSB modules are available.
Distributor ID:	Ubuntu
Description:	Ubuntu 18.04.1 LTS
Release:	18.04
Codename:	bionic
tcrha@hopenas:~$ which lsb_release
/usr/bin/lsb_release
```

**Comment by davestephens** [comment]
Doh, sorry - the actual file is lsb-release, so it should have been:
```
cat /etc/lsb-release
```
That said I can see you're on 18.04. Still a bit weird though, the module should "just work" - I'll do some digging.

**Comment by C-J1** [comment]
Hey folks. I found this myself when running the playbook on a "stock" Ubuntu 18.04.1 LTS. I added a step to the general tasks to install aptitude first, and that solved it. I'll raise a PR for it (my first ever PR, so please forgive if I screw it up :D)

**Comment by davestephens** [comment]
@C-J1 @thomascrha What ansible versions are you both using? It seems like this used to be a bug but is fixed in newer ansible version. Bug info: https://github.com/ansible/ansible/issues/30516

Please can you run ```ansible --version``` and paste the output?

**Comment by davestephens** [comment]
Fixed in #16
